src/main/java/com/sprint/mission/discodeit/entity/Channel.java

```diff
@@ -1,83 +1,91 @@
 package com.sprint.mission.discodeit.entity;
 
-import lombok.Getter;
-
 import java.io.Serial;
 import java.io.Serializable;
 import java.time.Instant;
 import java.util.List;
 import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
 
 //엔티티
 @Getter
+@Setter
+@AllArgsConstructor
 public class Channel implements Serializable {
-    //직렬화된 객체의 버전을 명시적으로 지정
-    @Serial
-    private static final long serialVersionUID = 1L;
-    //필드
-    private final UUID id; // 채널고유 id (내부 식별자)
-    private final Instant createdAt; // 생성시간
-    private Instant updatedAt; // 수정시간
-    private String title; // 채널 이름
-    private String description; // 채널 설명
-    private ChannelType channelType; // 음성채널 or 일반채널
-    private String ownerId;
-    private List<String> membersId;
 
+  //직렬화된 객체의 버전을 명시적으로 지정
+  @Serial
+  private static final long serialVersionUID = 1L;
+  //필드
+  private final UUID id; // = channelId, (내부 식별자)
+  private final Instant createdAt; // 생성시간
+  private Instant updatedAt; // 수정시간
+  private String title; // 채널 이름
+  private String description; // 채널 설명
+  private ChannelType channelType; // 음성채널 or 일반채널
+  private String ownerId;
+  private List<String> membersId;
+
+
+  //기본 생성자
+  //매개변수X
+  public Channel(UUID id, String name, String ownerId, ChannelType privateChannel,
+      List<String> membersId) {
+    this.id = UUID.randomUUID();
+    this.createdAt = Instant.now();
+    this.updatedAt = createdAt;
+  }
```

Comment:
id를 받았지만 무시하고 UUID를 별도로 생성해서 처리하고 있습니다. 제대로 처리 해주셔야합니다.
this.id = id 로 수정 필요

Reply:
추가적으로 
ChannelType privateChannel, 
List<String> membersId 
등은 무시되는 파라미터입니다. 받은것을 그대로 세팅하는게 생성자의 의도라고 생각됩니다.

```suggestion
public Channel(UUID id, String name, String ownerId, String description,
               ChannelType channelType, List<String> membersId) {
  this.id = id;
  this.title = name;
  this.ownerId = ownerId;
  this.description = description;
  this.channelType = channelType;
  this.createdAt = Instant.now();
  this.updatedAt = this.createdAt;
  this.membersId = membersId;
}
```